app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -247,6 +248,24 @@ def next_step_url
       idv_phone_url
     end
 
+    def save_resolution_vendors(form_response)
+      idv_session.resolution_vendor = form_response.extra.dig(
+        :proofing_results,
+        :context,
+        :stages,
+        :resolution,
+        :vendor_name,
+      )
+
+      idv_session.residential_resolution_vendor = form_response.extra.dig(
+        :proofing_results,
+        :context,
+        :stages,
+        :residential_address,
+        :vendor_name,
+      )
```

Comment:
Unimportant thought: obviously this is actually fine, but using form-handling code makes it _look_ like we're attributing costs based on user input. Not suggesting a change, though.